src/Slacklinker/Slack/ConversationsJoin.hs

```diff
@@ -1,93 +0,0 @@
-{-# LANGUAGE TemplateHaskell #-}
```

Comment:
I assume this got updated in slack-web?

Reply:
Yup:
- https://github.com/MercuryTechnologies/slack-web/commit/6a0e8d5e8986ca61b75b4c345c2dc1a38eb2cb46
- https://github.com/MercuryTechnologies/slack-web/commit/6c15d41f26cd5b19af34c5087e94591f056becbc

Needs changelog entries in that repo maybe? (Non-blocking)